addons/account_edi/models/account_edi_format.py

```diff
@@ -107,6 +108,15 @@ def _support_batching(self):
         # TO OVERRIDE
         return False
 
+    def _check_move_configuration(self, move):
```

Comment:
I'm not sure we need that in this backport

Reply:
But it's nice :) We don't _need_ it but since we're adding features, might as well do this small change, too.